### ISSUE TITLE
No focus indicator on focused dropdown item under windows high contrast mode

### ISSUE THREAD

**Issue by haonliu** [microsoft/fluentui#13828]
### Environment Information

- **Package version(s)**: 7.98.0
- **Browser and OS versions**: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit) 
Windows 10 Version 1909(OS Build 18363.418)​

### Describe the issue:

No focus indicator on focused dropdown item under windows high contrast mode.

### Please provide a reproduction of the issue in a codepen:

* Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown.
* Turn on windows high contrast black mode
* Use arrow keys to change selection on any of the dropdown components in the page.
* Check whether user can differentiate selected and non selected list items in the 'fast pass' drop down or not

#### Actual behavior:

There is no difference between selected and non selected "Fast Pass" dropdown list items in high contrast black/white theme
#### Expected behavior:

While navigating through already selected and non selected list items, there should be clear differentiation

**Comment by paulgildea** [comment]
@haonliu Thanks for the issue.

Adding this to #11742 mega issue.

@xugao - FYI

**Comment by xugao** [comment]
@haonliu - I do see the highlight background color when focused:
![image](https://user-images.githubusercontent.com/1207059/86188628-17110780-baf4-11ea-9f9b-1d3d4e568c63.png)

Could you provide a screenshot if you are not seeing the same?

Thanks
